src/cript/api/api.py

```diff
@@ -511,22 +510,42 @@ def save(self, project: Project) -> None:
 
         Returns
         -------
-        None
+        A set of extra saved node UUIDs.
             Just sends a `POST` or `Patch` request to the API
         """
+        if known_uuid is None:
+            known_uuid = set()
 
-        project.validate()
-
+        node.validate()
```

Comment:
I see we swapped project to node. I had a similar idea where I still have the same save for the project only, but there is a helper method that takes just a node and saves that to the API, so we can reuse the code within our brute force save method

Reply:
This is just a name here.

It doesn't change anything for the user.

It is just for devs. For a dev it would be confusing to call it project, since it can be other things, when called differently.

I am ok, wrapping that a little differently, such that we expose one with `project` to the user, but we can do it after we confirm this works.

Reply:
I agree, if we give the user to save only with project and we have a hidden one that works with node, that would be better. Yeah I agree we can do it after this works